### DISCUSSION TITLE
How to  show aside anchor link when using custom component?

### DISCUSSION THREAD

**Discussion by StreakingMan** [vuejs/vitepress#2950] [Q&A]
when I use h2 tag in custom component, It doesn't show aside anchor link:

Item.vue
``` vue.js
<template>
    <h2>{{title}}</h2>
    <p>{{desc}}</p>
</template>
```
list.vue
``` vue.js
<template>
    <Item v-for="(item, idx) in [{ title: 'hello', desc: 'word' }]" v-bind="item" :key="idx" />
</template>
```
article.md
``` markdown
<List />
```

I know that using v-for and # directly in Markdown can make it work:
``` article.md
<div v-for="(item, idx) in [{ title: 'hello', desc: 'word' }]" :key="idx" />

## {{ item.title }}

</div>
```

However, my scenario is complex, and I need to include the header within the custom component. Is there any API or workaround to implement this? 

Thanks!

**Comment by zennozenith** [comment]
I think this problem is answered here #1473
